.github/workflows/ci.yml

```
@@ -7,24 +14,41 @@ on:
       - release-*
 
 jobs:
-
-  pull:
+  # JOB to run change detection
+  changes:
+    runs-on: ubuntu-latest
+    # Set job outputs to values from filter step
+    outputs:
+      go: ${{ steps.filter.outputs.go }}
+      ui: ${{ steps.filter.outputs.ui }}
+    steps:
+      # For pull requests it's not necessary to checkout the code
+      - uses: dorny/paths-filter@v2
+        id: filter
+        with:
+          filters: |
+            go:
+              - makefile
```

```suggestion
              - Makefile
```

I found that `paths-filter` uses `picomatch` for matching files, and `picomatch` follows case-sensitive by default.